website/docs/language/resources/syntax.mdx

```diff
@@ -15,12 +15,17 @@ Each resource block describes one or more infrastructure objects, such
 as virtual networks, compute instances, or higher-level components such
 as DNS records.
 
+To see how Terraform manages resources when applying a configuration, see
+[Resource Behavior](/terraform/language/resources/behavior).
+
 ## Resource Syntax
 
-Resource declarations can include a number of advanced features, but only
-a small subset are required for initial use. More advanced syntax features,
-such as single resource declarations that produce multiple similar remote
-objects, are described later in this page.
+A "resource" block declares a resource of a specific type 
+with a specific local name. The name is used to refer to this resource 
+in the same Terraform module but has no meaning outside that module's scope.
+
+The resource type ("aws_instance") and name ("Web") together must be unique within a module because they
+serve as an identifier for a given resource.
```

Comment:
```suggestion
A `resource` block declares a resource of a specific type 
with a specific local name. Terraform uses the name when referring to the resource 
in the same module, but it has no meaning outside that module's scope.

In the following example,  the `aws_instance` resource type is named `web`. The resource type and name must be unique within a module because they serve as an identifier for a given resource.
```
A few more tweaks per our styles. In this case, use active voice and do not use quotation marks.